<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservierung</class>
 <widget class="QWidget" name="Reservierung">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neue Reservierung</string>
  </property>
  <widget class="QComboBox" name="comboBoxKunde">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonKundeDetail">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFahrzeugDetail">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAbbrechen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Fahrzeug:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>gültig bis:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUebernehmen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Übernehmen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFahrzeug">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kunde:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditGueltigBis">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEditGueltigBis</tabstop>
  <tabstop>comboBoxKunde</tabstop>
  <tabstop>buttonKundeDetail</tabstop>
  <tabstop>comboBoxFahrzeug</tabstop>
  <tabstop>buttonFahrzeugDetail</tabstop>
  <tabstop>buttonUebernehmen</tabstop>
  <tabstop>buttonAbbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
